docs/_pages/en/monitoring.md

```diff
@@ -16,16 +16,22 @@ These are common errors that apply globally and are the same for every URL path.
 | <code>401</code>        |   <code>Access token has expired</code>   | Access token has expired |
 | <code>403</code>   |  <code>Forbidden</code>   | Could not validate access token OR "Form ID" in request is not allowed to be accessed with the provided access token. |
 | <code>404</code> |  <code>Not found</code>   | Invalid request URL. |
+| <code>429</code> |  <code>Too many requests</code>   | Refer to API rate limit for the limit number. |
 | <code>500</code> |  <code>Internal server error</code>   | Internal error while serving request. |
 
-**Reporting errors and feedback**
+#### Reporting errors and feedback
 We expect there may be some additional features needed to get the data to target systems successfully. If you encounter an error, please submit a [technical support ticket](https://forms-formulaires.alpha.canada.ca/en/support) and we'll work with you to resolve the issue. Don't hesitate to include any feedback on the implementation so far — what works and what doesn't. Your feedback helps shape the product directly by informing our decisions on what issues are most important.
 
-### Refreshing API keys
+### API rate limit
 
-Refreshing an API key may become necessary if a key is compromised. Keys can be revoked and regenerated, simply requiring a change in the parameters being queried in the request.
+API throttling allows us to manage high API traffic and helps maintain the system’s stability. This helps avoid overloading situations where too many requests are processed simultaneously. 
+
+The limit for API requests from one form is **500 requests per minute**. Should you happen to exceed the limit, you will get a <code>RateLimitError</code>. You may either wait and try again, or request a higher limit for your form. If you require an API request rate increase, [contact Support](https://forms-formulaires.alpha.canada.ca/en/support).
 
-### Requesting a rate limit increase
+You can see the **current limit of requests** in <code>X-RateLimit-Limit</code>, the number of **remaining requests** in <code>X-RateLimit-Remaining</code>, and the moment **when the limit will reset** in <code>X-RateLimit-Reset</code>.
```

Comment:
We should tell them where to look for these information. They are part of the headers included in the API response.

We should also mention that when the limit has been reached there is an extra header key/value pair that will appear. `Retry-After` with a value in seconds that represent the time the client should wait before sending a new request.

Reply:
Excellent suggestion! Made a few commits with these changes.